draft-ietf-mops-streaming-opcons.md

```diff
@@ -751,6 +757,32 @@ For most of the history of the Internet, these transport protocols, described in
 
 For this reason, we have included a description of how the path characteristics that streaming media providers may see are likely to evolve over time.
 
+## Transport Protocols and Media Transport Protocols {#mtp}
+
+Within {{sec-trans}}, the term "Media Transport Protocol" is used to describe to describe the protocol of interest. This is easier to understand if the reader assumes a protocol stack that looks something like this:
+
+~~~~
+               Media
+    ---------------------------
+           Media Format
+    ---------------------------
+    Media Transport Protocol(s)
+    ---------------------------
+      Transport Protocol(s)
+~~~~
+
+where
+
+* "Media Format" would be something like an RTP payload format {{RFC2736}} or ISOBMFF {{ISOBMFF}},
+* "Media Transport Protocol" would be something like RTP or HTTP, and
+* "Transport Protocol" would be something like TCP or UDP.
+
+Not all possible streaming media applications follow this model, but for the ones that do, it seems useful to have names for "the protocol layers beteern Media and the Transport Layer".
+
+It is worth noting explicitly that the "Media Transport Protocol" layer might include more than one protocol. For example, a Media Transport Protocol might be defined to run over HTTP, or over WebTransport, over HTTP/3.
```

Comment:
s/over HTTP, or over WebTransport, over HTTP/3/over HTTP or over WebTransport
(no need to say HTTP twice)

Reply:
I was saying "WebTransport over HTTP", because my point was that there were two protocols occupying the Media Transport Layer", but you made me look at the text again, and there was more work for me to do. I also made it clear that the "Transport Protocol" layer could also include more than one protocol (QUIC over UDP), and I should have said "over HTTP", because WebTransport is also defined over HTTP/2. 

Again, thanks!